src/cc/heir.cpp

```diff
@@ -626,22 +626,24 @@ template <typename Helper> UniValue _HeirFund(int64_t txfee, int64_t amount, std
 
 		// CheckVinPubkey(mtx.vin, myPubkey, hasMypubkey, hasNotMypubkey);
 
+        /* why not?
```

Comment:
IMO: If the code does not apply, remove the code. It will exist in GitHub history, but not litter the code base.